scripts/mc_publish_sys_levels.py

```diff
@@ -19,6 +19,8 @@
     parser.add_argument('-r', '--revision', help="Specify revision or last/active/full/all for hpn.  [A]", default='A')
     parser.add_argument('-e', '--exact-match', help="Force exact matches on part numbers, not beginning N char. [False]",
                         dest='exact_match', action='store_true')
+    parser.add_argument('-f', '--force-new', dest='force_new', help="Force it to write a new hookup file.", action='store_true')
+    parser.add_argument('--force-specific', dest='force_specific', help="Force db and local cache file to use these values", action='store_true')
```

Comment:
Is one ever going to want to pass these arguments to this particular script? It doesn't seem like the right venue for fooling with what goes in the cache file.

Reply:
Maybe no force-specific, but possibly force-new.  This is going into the crontab and I liked the flexibility.